staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -466,7 +466,9 @@ func strategicPatchObject(
 ) error {
 	originalObjMap, err := runtime.DefaultUnstructuredConverter.ToUnstructured(originalObject)
 	if err != nil {
-		return err
+		return errors.NewInvalid(schema.GroupKind{}, "", field.ErrorList{
```

Comment:
it's unclear to me how an invalid error with an empty groupkind, a made-up field path, and no current value specified is an improvement... can you give an example of the difference in output as seen by an API client before/after this change?

Reply:
I have seen the above comment.

Let me spend some (spare) time on this ...

Reply:
If the error reporting is not improved, how about dropping this hunk and keep the above two in converter.go ?